lib/puppet/provider/package/pacman.rb

```diff
@@ -28,7 +27,10 @@ def self.yaourt?
   # Checks if a given name is a group
   def self.group?(name)
     begin
-      !pacman("-Sg", name).empty?
+      execpipe([command(:pacman), "-Sg"], failonfail=false) do |process|
+        output = process.readlines.join('').chomp!
```

Comment:
I think this should probably be simply `process.read.chomp!` and move the `empty?` to the result of `execpipe` (the blocks are generally expected to return strings not booleans).